akka-actor/src/main/scala/akka/actor/Props.scala

```diff
@@ -156,10 +112,9 @@ object Props {
  * }}}
  */
 @SerialVersionUID(2L)
-final case class Props(deploy: Deploy, clazz: Class[_], args: immutable.Seq[Any]) {
+final case class Props(deploy: Deploy = Props.defaultDeploy, clazz: Class[_], args: immutable.Seq[Any]) {
 
-  if (Modifier.isAbstract(clazz.getModifiers))
-    throw new IllegalArgumentException(s"Actor class [${clazz.getName}] must not be abstract")
+  Props.isAbstract(clazz)
```

Comment:
:-1: for making an `isSomething` method throw;

Seems this PR is not only "move" but some new methods?